src/components/Account/SubscriptionCard.tsx

```diff
@@ -29,23 +47,43 @@ export function SubscriptionCard() {
           <Title id="manage-subscription" order={2}>
             Membership
           </Title>
-          <Button
-            size="compact-sm"
-            radius="xl"
-            color="gray"
-            rightSection={<IconSettings size={16} />}
-            component={Link}
-            href="/user/membership"
-          >
-            Manage
-          </Button>
+          {showRedirectMessage ? (
+            <Button
+              size="compact-sm"
+              radius="xl"
+              rightSection={<IconExternalLink size={16} />}
+              onClick={handleRedirectToGreen}
+              className={`${greenClassNames?.btn} px-2 py-1 text-xs`}
```

Comment:
I just know the new eslint rule will yell for this 😛 let's use clsx instead

```suggestion
              className={clsx('px-2 py-1 text-xs', greenClassNames?.btn)}
```